### ISSUE TITLE
Conflict Between the CBO Implementation and the Definition of Original CK Paper

### ISSUE THREAD

**Issue by aaghamohammadi** [mauricioaniche/ck#50]
Hello,

I find a conflict between your implementation of CBO and the definition of the original CK paper. In the paper, authors say "CBO for a class is a count of the number of other classes to which it is coupled where two classes are coupled if methods declared in one class **use** methods or instance variables **defined** by the other class."

**Comment by mauricioaniche** [comment]
Yes. I think that's an outdated definition. To me, if you have a link between class A and class B, there's coupling. That can be realised by, e.g., defining a list of type A, but never invoking A. 

But one might want to know the coupling only about methods that are invoked in other classes. I'll rename this issue and leave it for someone who's willing to implement it!
